_posts/2024-02-18-how-its-made.md

```diff
@@ -0,0 +1,185 @@
+---
+title: >-
+  How the sausage is made, or: How OpenTTD is developed
+author: michi_cc
+---
+
+Ever wondered how a new OpenTTD release is made?
+How we decide what features to include and what to reject or how some people to seem to know the "future" before you?
+Curious what it means that OpenTTD is Open Source?
+Or maybe you've even wondered what it takes to get your idea included in OpenTTD?
+
+In this post, you'll get a peek behind the quite transparent curtain and learn more on how OpenTTD is developed.
+
+<!-- more -->
+
+## What's source anyway?
+
+Let's start with the meaning of Open Source, as the source (or source code) of a computer program really is the source of it all.
+In a nutshell, the source code of a computer program is the human-readable form of any program.
+For OpenTTD, we use the programming language [C++](https://en.wikipedia.org/wiki/C%2B%2B).
+This source code is then transformed by a piece of software called a compiler into a binary form that the processor in your device can run.
+The binary is then combined with various other data files to create the game you can download from our website or for example get via Steam.
+
+While every computer program has a source, not every program is Open Source.
+Open Source requires that the source code is freely available and allows modifications by other parties.
+You can look at the source code of OpenTTD right now in our [git repository on GitHub](https://github.com/OpenTTD/OpenTTD) <small>(but please come back here afterwards)</small>.
+To be able to track changes to the source code, we use a version control system called <code>git</code>.
+It shows us each and every individual change (called commit in git) that was made to the game since its inception 20 years ago.
```

Comment:
Suggest italicizing "commit" when it's treated as a "foreign word" in this way.

Reply:
Good idea with the italics here (and in the other places),